packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -999,6 +999,17 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   /** Unset value */
   'inputs.untyped-value.unset-item-button.text': 'Unset value',
 
+  /** Label for the insert menu all items filter  */
+  'insert-menu.filter.all-items': 'All',
+  /** Label for when the insert menu search shows no items */
+  'insert-menu.search.no-results': 'No items found',
+  /** Placeholder for the insert menu search field */
+  'insert-menu.search.placeholder': 'Search',
+  /** Tooltip for the insert menu grid view toggle */
+  'insert-menu.toggle-grid-view.tooltip': 'Toggle grid view',
+  /** Tooltip for the insert menu list view toggle */
+  'insert-menu.toggle-list-view.tooltip': 'Toggle list view',
```

Comment:
Is the insert menu going to be used outside of the array type? If not, I'd prefer that these are moved into the `inputs.array.*` prefix for clarity. Right now it is not obvious to translators what "insert menu" means.

Reply:
Really good question. I originally scoped these under `inputs.array.*`, but then @mariuslundgard and I started talking about how the menu could potentially be used in more places, so I changed the i18n scope to make it more generic. 

For example, here:
![Screenshot 2024-06-17 at 08 10 45](https://github.com/sanity-io/sanity/assets/4602382/8b22aad5-1ab9-491a-bb10-c522ccd04931)

But I agree, it should probably be under `inputs.array.*` for now.

Reply:
Adjusted the namespace now: 4c8175953534aca98d30077988231cb5179bff1d